### PR TITLE
PHP80 SCL from remi

### PR DESCRIPTION
**Pull request by stephdl**
https://github.com/NethServer/dev/issues/6356

This PR needs
- the remi-safe repository 
`yum install https://rpms.remirepo.net/enterprise/remi-release-7.rpm`
- the rpm `nethserver-remi-php80-php-fpm` the repository is at https://github.com/stephdl/nethserver-remi-php80-php-fpm

we have to find a way how to trade with the dependencies of php80 (SCL) from remi-safe

### CODE REVIEW

File: virtualhosts/etc/e-smith/templates/httpd/vhost-extra/15RhPhpScl_values [hunk 1]
```diff
@@ -12,10 +12,18 @@ if ($PhpVersion ne 'default')
...
+    if ($PhpVersion ne 'php80') {
+        $OUT .= << "EOF";
 <FilesMatch \.php\$>
   SetHandler "proxy:unix:/var/run/rh-$PhpVersion-php-fpm/$VhostName-$PhpVersion.sock|fcgi://localhost"
 </FilesMatch>
 EOF
+        } else {
+        $OUT .= << "EOF";
+<FilesMatch \.php\$>
+  SetHandler "proxy:unix:/var/run/$PhpVersion-php-fpm/$VhostName-$PhpVersion.sock|fcgi://localhost"
```
Comment (DavidePrincipi) [inline virtualhosts/etc/e-smith/templates/httpd/vhost-extra/15RhPhpScl_values]: Why not retaining the old `rh-` prefix here? The fragment name is `15RhPhpScl_values`... Apart from the socket path it seems there are no other differences with SCLo pacakges here.

Also the prop name is `PhpRhVersion`. Maybe retaining that "Rh" prefix requires less changes to the existing code.

File: api/lib/httpd_functions.pl [hunk 2]
```diff
@@ -33,6 +33,7 @@ sub list_features
         'rh-php71-php-fpm' => { installed => (-f '/etc/e-smith/db/configuration/defaults/rh-php71-php-fpm/type') ? JSON::true : JSON::false, packages => ['nethserver-rh-php71-php-fpm'] },
         'rh-php72-php-fpm' => { installed => (-f '/etc/e-smith/db/configuration/defaults/rh-php72-php-fpm/type') ? JSON::true : JSON::false, packages => ['nethserver-rh-php72-php-fpm'] },
         'rh-php73-php-fpm' => { installed => (-f '/etc/e-smith/db/configuration/defaults/rh-php73-php-fpm/type') ? JSON::true : JSON::false, packages => ['nethserver-rh-php73-php-fpm'] },
+        'php80-php-fpm' => { installed => (-f '/etc/e-smith/db/configuration/defaults/php80-php-fpm/type') ? JSON::true : JSON::false, packages => ['nethserver-remi-php80-php-fpm'] },
```
Comment (DavidePrincipi) [inline api/lib/httpd_functions.pl:36]: Even if not completely correct with the Remi package, I'd retain the `rh-` prefix: existing code looks more consistent with it and requires less changes. As alternative we could introduce a new repo prefix (say, `rm-`) specific to Remi.

```suggestion
        'rh-php80-php-fpm' => { installed => (-f '/etc/e-smith/db/configuration/defaults/rh-php80-php-fpm/type') ? JSON::true : JSON::false, packages => ['nethserver-remi-php80-php-fpm'] },
```

**Review by DavidePrincipi — COMMENTED** [review]
The "Rh" prefix is everywhere, also in DB props. We could act like it is an old heritage with no special meaning.

File: virtualhosts/etc/e-smith/templates/httpd/vhost-extra/15RhPhpScl_values (hunk 1 above)
Comment (stephdl) [reply virtualhosts/etc/e-smith/templates/httpd/vhost-extra/15RhPhpScl_values]: we use the same run directory, no condition inside the template

File: api/lib/httpd_functions.pl (hunk 2 above)
Comment (stephdl) [reply api/lib/httpd_functions.pl:36]: We cannot do this, the service name is `php80-php-fpm`

**Review by DavidePrincipi — APPROVED** [review]

**Pull request merged**